src/lib/tokenizer.ts

```diff
@@ -83,9 +84,7 @@ class Tokenizer {
   }
 
   static makeTokenizer(content: string, literals: Array<string>): void {
```

Comment:
so here I actually added a makeNewTokenizer parameter for the tests so that the guard/protection on makeTokenizer could be kept - let me know what you want to do! We could do either, I don't mind 😅

Reply:
I just got rid of the parameter, don't see a use case for calling the function but not wanting it to do anything :stuck_out_tongue: